### PR TITLE
Update troubleshooting-configuration.markdown

### PR DESCRIPTION
**Pull request by bcelary**
Added link to the logger component with a short description.

### CODE REVIEW

File: source/getting-started/troubleshooting-configuration.markdown [hunk 1]
```diff
@@ -41,6 +41,8 @@ sensor:
 
 Another common problem is that a required configuration setting is missing. If this is the case, the component will report this to `home-assistant.log`. You can have a look at [the various component pages](/components/) for instructions on how to setup the components.
 
+It may also be required to turn on debugging for a certain component. See [logger page](/components/logger/) for instructions on how to enable specific logging levels.
```
Comment (Landrash) [inline source/getting-started/troubleshooting-configuration.markdown]: To be consistent I would suggest:
>See the [logger](/components/logger/) components for on instructions on how to define the level of logging you require.

Comment (bcelary) [inline source/getting-started/troubleshooting-configuration.markdown]: Good point. See the latest commit if it's OK.

**Comment by Landrash** [comment]
Looks good. :)

**Comment by Landrash** [comment]
@fabaff Good to merge or any other additions?

**Pull request merged**